app/tests/e2e/test_tenants.py

```diff
@@ -200,17 +201,17 @@ async def test_create_tenant_ecosystem_verifier(
     assert_that(tenant).has_tenant_name(name)
     assert_that(tenant).has_created_at(wallet.created_at)
     assert_that(tenant).has_updated_at(wallet.updated_at)
-    assert wallet.settings["wallet.name"].startswith("ecosystem.")
+    assert_that(wallet.settings["wallet.name"]).is_type_of(str)
```

Comment:
Maybe (if I'm getting this correctly from above) this is uuid.hex so a string of certain format/lnegth. In which case we can check specifically for that instead of just string?!

Reply:
Can I address these in a follow up PR? I have a PR that is dependant on these changes so to avoid a lot of hassle it would be easier to address in that pr (and if so, could you approve)

Reply:
This has been addressed in #232